### ISSUE TITLE
bug: tsconfig include array triggers warning when path has leading ./

### ISSUE THREAD

**Issue by cam-narzt** [ionic-team/stencil#4667]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/ionic-team/stencil/issues) that already report this problem, without success.

### Stencil Version

4.0.4

### Current Behavior

if your tsconfig looks like this:

```
"include": [
        "./app/javascript"
    ],
```
you will get the very confusing warning:
```
[ WARN  ]  tsconfig.json "include" required
           In order for TypeScript to improve watch performance, it's recommended the "tsconfig.json" file should have
           the "include" property, with at least the app's "app/javascript" directory listed. For example: "include":
           ["app/javascript"]
```

### Expected Behavior

paths that are equivalent shouldn't raise this warning

### System Info

```shell
npx stencil info doesn't work on my system and isn't relevant in this case.
```

### Steps to Reproduce

clone linked repo
checkout `tsconfig_bug` branch
run `npm i`
run `npm run build`

### Code Reproduction URL

https://github.com/cam-narzt/stencilbug/tree/tsconfig_bug

**Comment by rwaskiewicz** [comment]
Hey @cam-narzt 👋 

Thanks for bug report + reproduction case! I took a look at it this morning, and wasn't able to reproduce the error exactly on my end. Following the steps to repro, I:
- Cloned the repo
- Checked out `tsconfig_bug` (where the HEAD is c417fd76254031f390d0b564c18af36d1dce350e)
- Ran `npm ci` to get the same dependencies
- Ran `npm run build`

I _do_ get an error message, but it looks like it's properly listing the 'src' directory in the error message instead of 'app/javascript':
```
➜  stencilbug git:(tsconfig_bug) npm run build

> demo-error@0.0.1 build
> stencil build --docs

[56:01.5]  @stencil/core
[56:01.7]  v4.0.4 🍧

[ WARN  ]  tsconfig.json "include" required
           In order for TypeScript to improve watch performance, it's recommended the "tsconfig.json" file should have
           the "include" property, with at least the app's "src" directory listed. For example: "include": ["src"]
```

Does it look like I'm missing anything here?

**Comment by rwaskiewicz** [comment]
Ah, gotcha! Makes sense - I can reproduce this now. Thanks!

**Comment by rwaskiewicz** [comment]
@cam-narzt I've created #4676 to fix this - I've created a dev build of Stencil that includes the fix atop of v4.0.4.  If you get a moment, can you try installing it and let me know how it goes?
```
npm i @stencil/core@v4.0.4-dev.1691704809.9fb61d7
```

**Comment by cam-narzt** [comment]
That fixed the issue on my end, lgtm.

**Comment by rwaskiewicz** [comment]
The fix for this issue went out with today's [v4.0.5 release](https://github.com/ionic-team/stencil/releases/tag/v4.0.5). As such, I'm going to close it out. Should it appear again, please feel free to open a new issue. Thanks!
